polkadot-parachains/statemine/src/lib.rs

```diff
@@ -498,6 +500,24 @@ pub type LocalAssetTransactor = CurrencyAdapter<
 	(),
 >;
 
+pub type FungiblesTransactor = FungiblesAdapter<
+	// Use this fungibles implementation:
+	Assets,
+	// Use this currency when it is a fungible asset matching the given location or name:
+	(
+		ConvertedConcreteAssetId<AssetId, Balance, AsPrefixedGeneralIndex<Local, AssetId, JustTry>, JustTry>,
+	),
+	// Do a simple punn to convert an AccountId32 MultiLocation into a native chain account ID:
+	LocationToAccountId,
+	// Our chain's account ID type (we can't get away without mentioning it explicitly):
+	AccountId,
+	// We only allow teleports of known assets.
+	NonZeroIssuance<AccountId, Assets>,
```

Comment:
This doesn't seem related to teleportation? It's checking whether the assets in the account has a non-zero issuance.

Reply:
It is. The `CheckAsset` generic is used to determine whether we track the issuance of a teleport (e.g. [here](https://github.com/paritytech/polkadot/blob/616667f14a165951d56d574c6ea77b22dd4ed5ec/xcm/xcm-builder/src/fungibles_adapter.rs#L166)). I use non zero total issuance as a proxy here. I'll adjust the comment correspondingly.

Reply:
Ah okay, you're trying to make the condition in `can_check_in` fail.

Reply:
For unknown assets, yes.